h2/src/docsrc/html/advanced.html

```diff
@@ -313,11 +313,8 @@ <h2 id="mvcc">Multi-Version Concurrency Control (MVCC)</h2>
 database waits until it can apply the change, but at most until the lock timeout expires.
 </p>
 <p>
-This feature is only available with the default MVStore storage engine.
-Changing the lock mode with it (<code>LOCK_MODE</code>) has no effect.
-</p>
-<p>
-MVCC is not used when using the PageStore storage engine.
+This feature is only available with the default MVStore storage engine,
+it is not used when using the PageStore storage engine.
```

Comment:
other way around, MVStore largely ignores it, PageStore is the thing that uses it

Reply:
PageStore does not have MVCC mode any more.

Previous section also created a false impression about table level locks. I moved MVCC section above it and placed necessary remarks about PageStore or MVStore-specific behaviors.